indexer/services/roundtable/src/tasks/update-compliance-data.ts

```diff
@@ -182,6 +183,19 @@ export default async function runTask(
       },
     );
 
+    const closeOnlyStatuses: ComplianceStatusFromDatabase[] = await
+    ComplianceStatusTable.findAll(
+      {
+        address: addressesToQuery,
+        status: [ComplianceStatus.CLOSE_ONLY, ComplianceStatus.BLOCKED],
+      },
+      [],
+    );
+    const closeOnlyAddresses: string[] = _.chain(closeOnlyStatuses)
+      .map(ComplianceDataColumns.address)
+      .uniq()
+      .value();
+
```

Comment:
Is it possible to exclude these addresses earlier than here? There's already been a query made to the compliance provider by this filter and the results for these addresses will then be un-used by still charged for.